styles.css

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
- [ ] Kindly give to the body `margin: 0;` in this way you can fix the white lines around your design

![image](https://user-images.githubusercontent.com/93630700/205456341-59876468-c148-49a4-ac59-927704884650.png)